src/FrameBuffer.cpp

```diff
@@ -557,7 +567,6 @@ void FrameBufferList::saveBuffer(u32 _address, u16 _format, u16 _size, u16 _widt
 
 	if (m_pCurrent != nullptr) {
 		bPrevIsDepth = m_pCurrent->m_isDepthBuffer;
-		m_pCurrent->m_readable = true;
 
```

Comment:
I'm using Android studio, it's doing some automatic fixing on its own. I need to pay closer attention to that. It seems to have removed m_readable due to it not being used.